### PR TITLE
Admin QOL fixes

### PR DESCRIPTION
**Pull request by NoooneyDude**
- Adds crew manifest list (separate from security records, which can be modified independently).
- Fixes admin respawn dropdown scroll sensitivity being very low.
- Sets the wizard gamemode to expected settings; no more Ragin' Mages.
- Ending the round as admin now closes the admin panel, so it no longer returns after the round restarts.
- Admins can now spawn spectators. Spectators themselves, however, still cannot respawn for metagaming reasons.
- Adds two serialized fields to `Occupation` to define an occupation's late-spawning behaviour (whether it spawns at arrivals or always at its unique point) and whether they are a crewmember, which determines whether they should be added to the crew manifest (and a security record generated, if not the cyborg or AI).
    - Likely fixes PDA crew manifest NRE.
    - Fixes `AutomatedAnnouncer` trying to announce fugitives.
- Admin-initiated round restarts are delayed by just five seconds instead of the default (60).

CL:Admin QOL fixes: respawning dropdown is no longer slow to scroll and admin-initiated round ending is fast.

### CODE REVIEW

File: UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+using UnityEngine.SceneManagement;
+
+namespace Systems
+{
+	/// <summary>
+	/// Contains a list of all mainstation crewmembers, including traitors and silicons but not, for example, fugitives or wizards.
+	/// </summary>
+	public class CrewManifestManager : MonoBehaviour
+	{
+		public static CrewManifestManager Instance;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs:13]: Codacy found an issue: [Change the visibility of 'Instance' or make it 'const' or 'readonly'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6384425)

File: UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+		/// </summary>
+		public List<SecurityRecord> SecurityRecords { get; private set; } = new List<SecurityRecord>();
+
+		#region Lifecycle
+
+		private void Awake()
+		{
+			if (Instance == null)
+			{
+				Instance = this;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs:31]: Codacy found an issue: [Make the enclosing instance method 'static' or remove this set on the 'static' field.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6384425)

**Comment by Bod9001** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 2
           

Complexity increasing per file
==============================
- UnityProject/Assets/Scripts/Objects/Telecomms/AutomatedAnnouncer.cs  1
- UnityProject/Assets/Scripts/UI/Items/PDA/GUI_PDACrewManifest.cs  1
- UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6384425&bid=20706465)

**Comment by Bod9001** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 2
           

Complexity increasing per file
==============================
- UnityProject/Assets/Scripts/UI/Items/PDA/GUI_PDACrewManifest.cs  1
- UnityProject/Assets/Scripts/Objects/Telecomms/AutomatedAnnouncer.cs  1
- UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6384425&bid=20706465)

File: UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+using UnityEngine.SceneManagement;
+
+namespace Systems
+{
+	/// <summary>
+	/// Contains a list of all mainstation crewmembers, including traitors and silicons but not, for example, fugitives or wizards.
+	/// </summary>
+	public class CrewManifestManager : MonoBehaviour
+	{
+		public static CrewManifestManager Instance;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs:13]: Codacy found an issue: [Change the visibility of 'Instance' or make it 'const' or 'readonly'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6384425)

File: UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+		/// </summary>
+		public List<SecurityRecord> SecurityRecords { get; private set; } = new List<SecurityRecord>();
+
+		#region Lifecycle
+
+		private void Awake()
+		{
+			if (Instance == null)
+			{
+				Instance = this;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs:31]: Codacy found an issue: [Make the enclosing instance method 'static' or remove this set on the 'static' field.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6384425)

**Comment by PerfectTangent** [comment]
- Admin-initiated round restarts are delayed by just five seconds instead of the default (60).

glory be, hooray!

**Pull request merged**